<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>368</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabMonitor">
      <attribute name="title">
       <string>Application</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QTabWidget" name="appTabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tabAppCommon">
          <attribute name="title">
           <string>Common</string>
          </attribute>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>NordVPN:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="AppPathEditor" name="leNVPNPath" native="true"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Check Interval:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="spinBoxInterval">
             <property name="suffix">
              <string>s</string>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>3600</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Log lines limit:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="spinBoxLogLines">
             <property name="maximum">
              <number>10000</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QCheckBox" name="checkBoxAutoActive">
             <property name="text">
              <string>Activate on app start</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabAppMap">
          <attribute name="title">
           <string>Map</string>
          </attribute>
          <layout class="QVBoxLayout" name="mapVerticalLayout">
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabTray">
          <attribute name="title">
           <string>Tray</string>
          </attribute>
          <widget class="QGroupBox" name="groupBoxMessage">
           <property name="geometry">
            <rect>
             <x>9</x>
             <y>9</y>
             <width>260</width>
             <height>120</height>
            </rect>
           </property>
           <property name="title">
            <string>Message balloon</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Duration:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="cbIgnoreFirstConnected">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Ommit first &quot;Connected&quot;</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QCheckBox" name="checkBoxMessagePlainText">
              <property name="text">
               <string>Plain text</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="spinBoxMsgDuration">
              <property name="suffix">
               <string>s</string>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>3600</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QGroupBox" name="groupBox">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>180</y>
             <width>451</width>
             <height>161</height>
            </rect>
           </property>
           <property name="title">
            <string>Icon</string>
           </property>
           <widget class="QLineEdit" name="lineEdit">
            <property name="geometry">
             <rect>
              <x>210</x>
              <y>40</y>
              <width>113</width>
              <height>23</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label_5">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>40</y>
              <width>57</width>
              <height>15</height>
             </rect>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
           <widget class="QToolButton" name="toolButton">
            <property name="geometry">
             <rect>
              <x>330</x>
              <y>40</y>
              <width>28</width>
              <height>22</height>
             </rect>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Actions</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="ActionsTab" name="tabActionsYangl">
          <attribute name="title">
           <string>yangl</string>
          </attribute>
         </widget>
         <widget class="ActionsTab" name="tabActionsNordVPN">
          <attribute name="title">
           <string>NordVPN</string>
          </attribute>
         </widget>
         <widget class="ActionsTab" name="tabActionsUser">
          <attribute name="title">
           <string>Custom</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ActionsTab</class>
   <extends>QWidget</extends>
   <header>actionstab.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AppPathEditor</class>
   <extends>QWidget</extends>
   <header>apppatheditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
